src/main/scala/com/itechart/ny_accidents/utils/NumberUtils.scala

```diff
@@ -0,0 +1,11 @@
+package com.itechart.ny_accidents.utils
+
+object NumberUtils {
+  def validateDouble(number: Double): Option[Double] = {
```

Comment:
Do we still need to utils object? I guess it can be removed together with appropriate test

Reply:
Removed